src/com/intellij/idea/plugin/hybris/project/descriptors/YModuleLibDescriptorUtil.kt

```diff
@@ -48,6 +49,7 @@ object YModuleLibDescriptorUtil {
     fun addRootProjectLibrary(
```

Comment:
looks like this method is only responsible for creation of the `BACKOFFICE_LIBRARY_GROUP`, feel free to rename and refactor accordingly